openlibrary/templates/account/create.html

```diff
@@ -57,21 +57,16 @@ <h1>$_("Sign Up")</h1>
 
         <hr/>
 
-        $if form.has_recaptcha:
-            <div class="formElement">
-                <div class="label smaller lighter">$_("If you have security settings or privacy blockers installed, please disable them to see the reCAPTCHA.")</div>
-                <div class="g-recaptcha" data-sitekey="$form['recaptcha'].public_key"></div>
```

Comment:
Do we need to update all other uses of this? `g-recaptcha` is used in a few places

Reply:
Tested, I think it's good as is!